backend/models/loginRecord.js

```diff
@@ -0,0 +1,127 @@
+/**
+ *	Copyright (C) 2014 3D Repo Ltd
+ *
+ *	This program is free software: you can redistribute it and/or modify
+ *	it under the terms of the GNU Affero General Public License as
+ *	published by the Free Software Foundation, either version 3 of the
+ *	License, or (at your option) any later version.
+ *
+ *	This program is distributed in the hope that it will be useful,
+ *	but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *	GNU Affero General Public License for more details.
+ *
+ *	You should have received a copy of the GNU Affero General Public License
+ *	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ "use strict";
+ const db = require("../handler/db");
+ 
+ //detects edge as browser but not device
+ var uaParserJs = require('ua-parser-js');
```

Comment:
use `const`.

Always `const` unless you have to reassign them.

Var is discouraged for ES6+ javascript. You shouldn't really find yourself needing to use var anymore. It has some weird scoping definitions which you want to avoid https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/